client/app/resources/mock-data.factory.js

```diff
@@ -0,0 +1,431 @@
+(function() {
+  'use strict';
+
+  angular.module('app.resources')
+    .factory('MockData', MockDataFactory);
+
+  /** @ngInject */
+  function MockDataFactory() {
+    var service = {
+      retrieveMockData: retrieveMockData
+    };
+
+    return service;
+
+    function retrieveMockData(collection) {
+      if (collection === 'service_catalogs') {
```

Comment:
Hi, I do in fact have those REST api's in place in the code, but I choose to mock them anyway because I do not have a working database in my dev environment.  To switch to REST api's, just need to remove the 'mock' option in the CollectionsApi call, ex:

```
function resolveServiceCatalogs(CollectionsApi) {
   var options = {
   mock: true,       <---- remove this
   expand: 'resources',
   sort_by: 'name',
   sort_options: 'ignore_case'};

   return CollectionsApi.query('service_catalogs', options);
}
```